Makefile

```diff
@@ -179,6 +179,9 @@ PYTHON     ?= $(VENV)/bin/python3
 # preprocessor for this because it won't substitute inside string literals.
 SPEC_REPLACE_VARS := sed -e 's|$$(BUILD_DIR)|$(BUILD_DIR)|g'
 
+# Audio tools
+AUDIO_EXTRACT := $(PYTHON) tools/audio/extraction/audio_extract.py
```

Comment:
It seems weird that this is the only tools/foo.py that gets its own variable, especially now that the path is not that long. I think inlining it would be slightly more aesthetically pleasing in `make setup`

Reply:
It's mostly for symmetry with the other audio tools, since it's going to end up like this later:
```make
# Audio tools
AUDIO_EXTRACT := $(PYTHON) tools/audio_extraction.py
SAMPLECONV    := tools/audio/sampleconv/sampleconv
SBC           := tools/audio/sbc --matching
SFC           := tools/audio/sfc --matching
ELFPATCH      := tools/audio/elfpatch
ATBLGEN       := tools/audio/atblgen
AFILE_SIZES   := tools/audio/afile_sizes
```

Reply:
The $(PYTHON) one looks like the odd one out there too, but okay